vagrant/provisioning/roles/saml-config/tasks/main.yml

```diff
@@ -52,20 +52,21 @@
     replace: "{{ item.replace }}"
   loop:
     - regexp: 'arkcase/acm/private/saml-keystore'
-      replace: 'arkcase/acm/private/keystore'
+      replace: 'arkcase/acm/private/arkcase.ks'
     - regexp: 'type="java.lang.String" value="password"'
-      replace: 'type="java.lang.String" value="changeit"'
+      replace: 'type="java.lang.String" value="{{ internal_host }}"'
```

Comment:
In AWS sites I use the local host name for both the key name and also the key store password.  May need to refactor this to use other / new facts.